tiledb/sm/c_api/tiledb.h

```diff
@@ -4316,7 +4367,10 @@ TILEDB_EXPORT int32_t tiledb_query_get_array(
  *
  * @note The stride is currently unsupported. Use `nullptr` as the
  *     stride argument.
+ *
+ * @note This is a TILEDB_DEPRECATED api.
```

Comment:
Remove those and use TILEDB_DEPRECATED_EXPORT

Reply:
I was previously instructed to do it this way, I think because of problems it causes tiledb internal clients.

Reply:
Look at Beka's latest changes for the with key API. Looks like we are using TILEDB_DEPRECATED_EXPORT now.

Reply:
I do not see anywhere that Beka has used TILEDB_DEPRECATED_EXPORT in tiledb.h for the _with_key changes.
........
Looking at Beka's changes for the _with_key APIs in tiledb.h, commit 5658e01d24a70c,
I see only added comment lines similar to what I have here, (one example, " * This is a deprecated API.")
not active deprecation as you are requesting.
[search for the diff of tiledb/sm/c_api/tiledb.h at following link to see the comments added...
https://github.com/TileDB-Inc/TileDB/compare/98c222cb7c63..5658e01d24a7]
(A later commit 3b5128a2df7 for the same effort by Beka has no further changes to tiledb.h.)
(She does have active deprecations in cpp_api headers, but those apparently do not have the same indirect undesireable noise generation as do active deprecations in tiledb.h.)

Reply:
Sorry I meant https://github.com/TileDB-Inc/TileDB/pull/2399/